presto-main/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -572,6 +573,21 @@ public ColumnMetadata getColumnMetadata(Session session, TableHandle tableHandle
         return ImmutableList.copyOf(tables);
     }
 
+    @Override
+    public List<QualifiedObjectName> getMaterializedViews(Session session, QualifiedObjectName tableName)
+    {
+        requireNonNull(tableName, "tableName is null");
+
+        Optional<CatalogMetadata> catalog = getOptionalCatalogMetadata(session, tableName.getCatalogName());
+        if (catalog.isPresent()) {
+            ConnectorMetadata metadata = catalog.get().getMetadata();
+            ConnectorSession connectorSession = session.toConnectorSession(catalog.get().getConnectorId());
+            List<SchemaTableName> materializedViews = metadata.getMaterializedViews(connectorSession, toSchemaTableName(tableName));
+            return ImmutableList.copyOf(materializedViews.stream().map(convertFromSchemaTableName(tableName.getCatalogName())).collect(Collectors.toList()));
```

Comment:
toImmutableList, if we are already building an immutable list, there is no need to call ImmutableList.copyOf